### PR TITLE
London Class 7 - Negin Derakhshan - HTML/CSS - Week 2

### PR DESCRIPTION
**Pull request by NeginDn**
# Your Details

- Your Name: Negin Derakhshan
- Your City: London
- Your Slack Name: Negin Dn

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,10 +12,65 @@
...
+    <section>
+        <div id="container-heading">
+                <p>Every one needs a little Karma</p>
+        </div>
+
+        <div id="container-items">
+            <section><img class="container-icons" src="img/icon-devices.svg" alt="Computer, tablet and cellphone icon"><p class="container-text">Internet for all devices</p></section>
+            <section><img class="container-icons" src="img/icon-coffee.svg" alt="cup of coffee icon"><p class="container-text">Boost your productivity</p></section>
+            <section><img class="container-icons" src="img/icon-refill.svg" alt="gass container logo"><p class="container-text"> Pay as you go</p></section>
+        </div>
```
Comment (Behnazz) [inline index.html:54]:
```suggestion
  <div id="container-items">
          <section>
            <img
              class="container-icons"
              src="img/icon-devices.svg"
              alt="Computer, tablet and cellphone icon"
            />
            <p class="container-text">Internet for all devices</p>
          </section>
          <section>
            <img
              class="container-icons"
              src="img/icon-coffee.svg"
              alt="cup of coffee icon"
            />
            <p class="container-text">Boost your productivity</p>
          </section>
          <section>
            <img
              class="container-icons"
              src="img/icon-refill.svg"
              alt="gass container logo"
            />
            <p class="container-text">Pay as you go</p>
          </section>
        </div>
```

File: index.html [hunk 2]
```diff
@@ -12,10 +12,65 @@
...
 
+    </section>
+    </main>
+    <hr>
+    <footer>
+        <section>
+             <div id="footer-heading">
+                 <p>Join us on</p>
+            </div>
+
```
Comment (Behnazz) [inline index.html:64]:
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -12,10 +12,65 @@
...
+        <section>
+             <div id="footer-heading">
+                 <p>Join us on</p>
+            </div>
+       
+         <div id="footer-item">
+             <section><img class="footer-icons" src="img/twitter-icon.svg" alt="  "></section>
+             <section><img class="footer-icons" src="img/facebook-icon.svg" alt="  "></section>
+             <section><img class="footer-icons" src="img/instagram-icon.svg" alt="  "></section>
+        </div>
```
Comment (Behnazz) [inline index.html:69]:
```suggestion
             <div id="footer-item">
          <section>
            <img
              class="footer-icons"
              src="img/twitter-icon.svg"
              alt="twitter-logo"
            />
          </section>
          <section>
            <img
              class="footer-icons"
              src="img/facebook-icon.svg"
              alt="facebook-logo"
            />
          </section>
          <section>
            <img
              class="footer-icons"
              src="img/instagram-icon.svg"
              alt="instagram-logo"
            />
          </section>
        </div>
```

**Review by Behnazz — COMMENTED** [review]
Welldone Negin. 
It is good that you used flex. Just some minor comments, beside the indentation correction :)

File: index.html [hunk 4]
```diff
@@ -12,10 +12,103 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header id= "header"><!--The top part of the page-->
+        <img id="header-img" class="logo" src="karma-logo.svg"  alt= "Karma's logo, Orange background with white element"><!-- Karma logo-->
```
Comment (louisechow) [inline index.html:19]: Don't forget to close the image tag

File: index.html [hunk 5]
```diff
@@ -12,10 +12,103 @@
...
+                <li><a href="#store">Store</a></li>
+                <li><a href="#blog">Blog</a></li>
+                <li><a href="#help">Help</a></li>
+                <li><a href="#login">Login</a></li>
+            </ul>
+            </nav>
+    </header>
+    <main>
+        <section>
+            <img id="bg-img" src="img/first-background.jpg" alt="Someone's half body and a tablet on table">
```
Comment (louisechow) [inline index.html:33]: This image tag needs to be closed.

File: index.html [hunk 6]
```diff
@@ -12,10 +12,103 @@
...
+    </section>
+    <section> 
+        <button class="intro-button">Learn More</button>
+    </section> 
+
+    <section>
+        <div id="container-heading">
+                <p>Every one needs a little Karma</p>
+        </div>
+
```
Comment (louisechow) [inline index.html:49]: I think this section is missing a closing tag...

File: index.html [hunk 7]
```diff
@@ -12,10 +12,103 @@
...
+    </main>
+    <hr>
+    <footer>
+        <section>
+             <div id="footer-heading">
+                 <p>Join us on</p>
+            </div>
+             <div id="footer-item">
+          <section>
+            <img
```
Comment (louisechow) [inline index.html:87]: Good to see that this group of image tags are closed 👍

My preference would be to put the image tag all on one line. I think it's easier to read that way.
E.g. twitter-logo

If you have a lot of attributes in your tag, then you should definitely split it into multiple lines. But this only has 3 short attributes.

File: index.html [hunk 8]
```diff
@@ -12,10 +12,103 @@
...
+            />
+          </section>
+        </div>
+    </section>
+    <section class="footer-copyright">
+        <p> &copy Karma Mobility,Inc. </p>
+    </section>
+    </footer>
 </body>
 </html>
```
Comment (louisechow) [inline index.html:114]: Overall, very good use of semantic tags such as `<main>`, `<section>`, `<footer>` etc.

File: css/style.css [hunk 9]
```diff
@@ -16,4 +14,116 @@ body {
...
+.footer-icons {
+  width: 25px;
+  margin-right: 20px;
+  text-align: center;
+}
+.footer-copyright {
+  font-size: 15px;
+  text-align: center;
+  color: rgb(119, 119, 119);
+}
```
Comment (louisechow) [inline css/style.css:129]: This is great! 🌟

**Comment by louisechow** [comment]
Great job @NeginDn! Your page looks great and I'm happy to see you figured out how to center align the text for your icons. ☺️

One thing that would be great for you to do would be to go and make your indentation consistent. Indentation is important because it makes the code easier to read and you can spot mistakes. 

There were a few tags that you forgot to close and I think if your indentation was more consistent it would be easier for you to spot this.

**Pull request closed without merging**